hardware/setup/create_framework.tcl

```diff
@@ -96,7 +96,11 @@ if { $hls_support == "TRUE" } {
 }
 set_property used_in_simulation false [get_files $hdl_dir/core/psl_fpga.vhd]
 # Action Files
-add_files            -fileset sources_1 -scan_for_includes $action_dir/
+if { $hls_support == "TRUE" } {
+  add_files            -fileset sources_1 -scan_for_includes $action_dir/vhdl/
```

Comment:
The variable `$action_dir` should point to the directory containing the HDL sources for the action. That usually does not contain a subdirectory vhdl.